include/zephyr/bluetooth/l2cap.h

```diff
@@ -557,6 +558,27 @@ int bt_l2cap_ecred_chan_connect(struct bt_conn *conn,
  */
 int bt_l2cap_ecred_chan_reconfigure(struct bt_l2cap_chan **chans, uint16_t mtu);
 
+/** @brief Reconfigure Enhanced Credit Based L2CAP channels
+ *
+ *  Experimental API to reconfigure with explicit MPS and MTU values.
+ *
```

Comment:
@kartben do we have any good way of indicating that a specific function should be considered experiemental? Or just like this?
Is `@version` something we use for functions, or just API groups?